policies/registry.rego

```diff
@@ -0,0 +1,42 @@
+package before_resolution
+
+# This file enforces policies requiring all attributes to be defined within
+# a semantic convention "registry".  This is a naming/structure convention
+# used by semantic conventions.
+
+# Helper to create attribute registry violations.
+attr_registry_violation(violation_id, group_id, attr_id) = violation {
+	violation := {
+		"id": violation_id,
+		"type": "semconv_attribute",
```

Comment:
also not sure about type - I assume we could do cross-signal checks (metric collision with attribute name), id characters, etc. 
What would be the benefit of having the classification? 

(I know it's required property, but I think it shouldn't be)

Reply:
See above discussion.  We're going to spend some time fixing violation requirements in weaver, either going more flexible or *simpler* on what's included so you can do more in Rego.